src/dom-to-image-more.js

```diff
@@ -1018,7 +1007,8 @@
         const defaultElement = document.createElement(tagName);
         sandbox.contentWindow.document.body.appendChild(defaultElement);
         // Ensure that there is some content, so that properties like margin are applied.
-        defaultElement.textContent = '.';
+        // we use zero-width space to handle FireFox adding a pixel
+        defaultElement.textContent = '\0x200B';
```

Comment:
Not sure if this is better or worse, but as long as we're not trying to inject `SCRIPT` tags, it shouldn't matter anymore :)

Reply:
To get the string working in my console:
```suggestion
        defaultElement.textContent = '\u200b';
```

Generally I think the ZWSP is better:
- we're now thinking about "empty" elements for `getDefaultStyle`
- the hack now narrowly targets the bug
- plus for free, we avoid a value collision when an element's CSS has the precise width of a `.` but is empty

Reply:
Applied this in another branch, thanks!